CHANGELOG.rst

```diff
@@ -2,6 +2,142 @@
 Release Notes
 =============
 
+.. _Release Notes_7.63.0:
+
+7.63.0
+======
+
+.. _Release Notes_7.63.0_Prelude:
+
+Prelude
+-------
+
+Release on: 2025-02-19
+
+- Please refer to the `7.63.0 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7630>`_ for the list of changes on the Core Checks
+
+
+.. _Release Notes_7.63.0_Upgrade Notes:
+
+Upgrade Notes
+-------------
+
+- Bump the Python version to 3.12.8
+
+
+.. _Release Notes_7.63.0_New Features:
+
+New Features
+------------
+
+- Add the capability in ``kubernetes_state_core`` check to collect metrics for Kubernetes custom resources
```

Comment:
Should this be moved to the DCA changelog too? Modifies `corechecks/cluster` https://github.com/DataDog/datadog-agent/pull/31715